src/App.tsx

```diff
@@ -1,13 +1,22 @@
 import React from "react";
 
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
 
-import { Home, Layout, List, ManageList } from "./views";
```

Comment:
Last piece of "food for thought" here comes from experience in working in very large React apps: eventually the directory structure of directories like "views" can be very confusing. I find a great solution to this is to have a directory like "views" have two sub directories: something like "authenticated" and "unauthenticated". Our `List`, `ManageList`, and others could live in the new `authenticated` directory, and the new `PageNotFound` and potentially others could live in the `unauthenticated` directory

When making changes like that, there's two approaches: one is preferred, but sometimes the other is unavoidable. The preferred option here, that would work well for this work, would be to create a new PR that is branched off of this one (or wait until this merges and branch it off of main), and do the file movements in that PR. Something important with this path would be to *only* move the files, don't change anything else. This allows you to say in your MR "just moving things around" , which makes it infinitely easier for reviewers, as well as your teammates who will likely have to deal with merge conflicts.

The other solution would be to do the changes here, which I would advise you _always_ try to avoid because it can make the diffs seem much larger than they really are and make reviewing the "real" changes harder.

Reply:
This is really good info. I was starting to feel like maybe "reorganization" was on the forefront, but with more files in play I didn't think it was something that would typically be attached to a PR where code has been changed. So knowing that there is multiple approaches, and a preferred method, is really nice. 

I'm gonna take the create PR off of this one and move because I just also prefer that. It seems cleaner and easier to deal with.